diff -up lyx-2.3.2/src/support/FileName.cpp.omv~ lyx-2.3.2/src/support/FileName.cpp
--- lyx-2.3.2/src/support/FileName.cpp.omv~	2018-12-20 07:06:16.796052901 +0100
+++ lyx-2.3.2/src/support/FileName.cpp	2018-12-20 07:06:43.870444088 +0100
@@ -74,7 +74,9 @@ using namespace lyx::support;
 // understand those weak symbols (seen on HP-UX, tru64, AIX and
 // others). Thus we force an explicit instanciation of this particular
 // template (JMarc)
-template struct boost::detail::crc_table_t<32, 0x04C11DB7, true>;
+#if 0
+template struct boost::detail::crc_table_t<32, CHAR_BIT, 0x04C11DB7, true>;
+#endif
 
 namespace lyx {
 namespace support {
